services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue

```diff
@@ -1,7 +1,59 @@
 <template>
   <form @submit.prevent="handleSubmit(!v$.$invalid)">
+    <div v-if="!invitation_url">
+      <!-- Invitation type -->
+      <div class="mb-3">
+        <span>{{ $t('connect.invitation.type') }}</span>
+        <div class="my-2">
+          <RadioButton
+            v-model="isOob"
+            input-id="type1"
+            name="type"
+            :value="false"
+          />
+          <label for="type1" class="ml-2">{{
+            $t('connect.invitation.typeConnections')
+          }}</label>
+        </div>
+        <div>
```

Comment:
This extra div needed? The other group doesn't have one. 
It's only a tiny bit of code but these 2 elements consisting of title+radio+label+radio+radio look like they could be the same component.

Reply:
The divs around the Radio + Label elements? Without those things end up in-line
![image](https://github.com/bcgov/traction/assets/17445138/f8b58a90-31f3-49a9-b2cb-3b09fc05a776)

I might be misunderstanding

Reply:
Oh, I see. Just the top one needed a margin.